scripts/ci/nodejs.sh

```diff
@@ -4,6 +4,14 @@ srcdir=`dirname $0`
 test -z "$srcdir" && srcdir=.
 . ${srcdir}/common.sh
 
-DEFAULT_COMPOSE_ARGS="master --no-apm-server-dashboards --no-apm-server-self-instrument --no-kibana --with-agent-nodejs-express --force-build --build-parallel"
+# github;version -> npm install elastic/apm-agent-nodejs#version
+# release;version -> npm install elastic-apm-node@version
+if [ ! -z "${APM_AGENT_NODEJS_PKG}" ]; then
+  APM_AGENT_NODEJS_PKG=${APM_AGENT_NODEJS_PKG/'github;'/'elastic/apm-agent-nodejs#'}
```

Comment:
this is a lot like but not identical to https://github.com/elastic/apm-integration-testing/blob/d6103cb90c1b5960220ceb45a1f018332a80087e/scripts/ci/versions_nodejs.sh - what's the plan here?

Reply:
The differences between `.sh` and `version_*.sh` scripts are some agent related parameters, this means every time we change some parameter in `.sh` scripts, we have also to change it on `version_*.sh` scripts, Do we need to duplicate scripts for some reason?

Reply:
No reason to duplicate if you can unify them, I'm just trying to understand the reasoning as this pull request does a bunch of things at once.  I think it would help a lot if you could pull these changes into smaller, focused pull requests.

Reply:
>No reason to duplicate if you can unify them, I'm just trying to understand the reasoning as this pull request does a bunch of things at once and doesn't remove the current version scripts.

those scripts are used by the current CI, so I can not remove them yet.

>I think it would help a lot if you could pull these changes into smaller, focused pull requests.

sure, I will split it into smaller pieces